src/main/kotlin/com/atlassian/performance/tools/jiraactions/w3c/VerboseJsonFormat.kt

```diff
@@ -26,7 +27,10 @@ internal class VerboseJsonFormat {
                 .map { deserializeNavigationTiming(it) },
             resources = getJsonArray("resources")
                 .map { it.asJsonObject() }
-                .map { deserializeResourceTiming(it) }
+                .map { deserializeResourceTiming(it) },
+            elements = getJsonArray("elements")
+                ?.map { it.asJsonObject() }
+                ?.map { deserializeElementTiming(it) }
```

Comment:
Indeed, `getJsonArray("elements")` could be `null`, therefore the null-safe dereference is needed: `?.`
That's because of old JSON files, which should remain parseable for compatibility.

But if we set the `elements` property to be non-null for consistency, then we need to default to an empty list (via the Elvis operator: `?:`):
```suggestion
            elements = getJsonArray("elements")
                ?.map { it.asJsonObject() }
                ?.map { deserializeElementTiming(it) }
                ?: emptyList()
```

Reply:
- [ ] Do it!